tests/tools/read-storage/test_read_storage.py

```diff
@@ -134,3 +85,50 @@ def test_read_storage(web3, ganache, solc_binary_path) -> None:
                 f.write(str(change.t2))
 
     assert not diff
+
+
+# pylint: disable=too-many-locals
+@pytest.mark.usefixtures("web3", "ganache")
+def test_unstructured_storage(web3, ganache) -> None:
+    assert web3.is_connected()
+    bin_path = Path(TEST_DATA_DIR, "UnstructuredStorageLayout.bin").as_posix()
+    abi_path = Path(TEST_DATA_DIR, "UnstructuredStorageLayout.abi").as_posix()
+    bytecode = get_source_file(bin_path)
+    abi = get_source_file(abi_path)
+    contract = deploy_contract(web3, ganache, bytecode, abi)
+    contract.functions.store().transact({"from": ganache.eth_address})
+    address = contract.address
+
+    sl = Slither(Path(TEST_DATA_DIR, "unstructured_storage-0.8.10.sol").as_posix())
```

Comment:
This is missing `solc_binary_path` which handles installing the solc dependency. The code should look something like https://github.com/crytic/slither/blob/4149f6034163e6252a6a277ccd03d811701ec757/tests/tools/read-storage/test_read_storage.py#L93-L105